scripts/supervisord/assemble-and-restart

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -x
+set -eo pipefail
+
+# Override PATH for the assemble script so that /usr/bin precedes
+# /opt/app-root/bin
+
+# PATH=/usr/bin:$PATH
+
+# Override HOME directory for the assemble script so that per user
+# files are not installed under /opt/app-root/src.
+
+# HOME=/opt/app-root
+
+# For some reason, SOMEWHERE, $PATh is being overriden.
+# TODO: add script to find in /opt/rh all the *bin* folders and add that to $PATH
+for i in $(find /opt/rh/ -name 'bin' -type d | grep '/root/usr/bin'); do
+        PATH=$i:$PATH
+done
+
+# s2i fails when /tmp/src/ is empty
+mkdir -p /tmp/src
+touch /tmp/src/.dummy
+
+# run s2i assemble script from source directory if it has been overridden  there
+if [ -f /opt/app-root/src/.s2i/bin/assemble ]; then
+    /opt/app-root/src/.s2i/bin/assemble
+else
+    /usr/libexec/s2i/assemble
+fi
+
+# restart run process
+/var/lib/supervisord/bin/supervisord ctl reload
```

Comment:
This just reload the supervisord process, not the application that is supervisord "supervising"

You will probably need to run `ctl stop` and `ctl start`